source/partials/_nav.slim

```diff
@@ -5,8 +5,6 @@ nav.main-nav
     li
       a href="/gems" Gems
     li
-      a href="http://github.com/dry-rb" GitHub
+      a href="/community" Community
     li
-      a href="https://discuss.dry-rb.org" Discuss
```

Comment:
Makes sense. Links in the footer sounds good too.

On 7 August 2016 at 05:57:51, Tim Riley (notifications@github.com) wrote:

> In source/partials/_nav.slim
> <https://github.com/dry-rb/dry-rb.org/pull/93#discussion_r73795387>:
>
> >      li
> > -      a href="https://discuss.dry-rb.org" Discuss
>
> Yes, I figured it would be better to have a simpler top-level nav, and we
> link to all the community interactions points (including the forum) over on
> the community page anyway.
>
> I think there's a couple of things we could do to raise the prominence of
> links to the forum:
>
>    - Make the links in the top paragraph on the community page bigger,
>    bolder and more obvious. Start by just bolding them?
>    - Build out an extra nav in the footer that can link to all these
>    specific items (perhaps have a simple, vertical list of links replace the
>    chunky github/twitter/gitter "buttons")
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dry-rb/dry-rb.org/pull/93/files/d2f2e4bf23ff258bf73b980f8a395c11f1104d29#r73795387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAEKq-CA3T9hzUpvG2XHQrD8kuiQttiks5qdVe_gaJpZM4JcPih>
> .
>